product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestBase.java

```diff
@@ -269,6 +281,17 @@ public void deleteRole(String role, String adminUsername, String adminPassword)
         }
     }
 
+    public void getUserList(String username, String password, String[] roleList,
```

Comment:
this method doesn't return anything. but the name suggests as it is. Also the exception message is confusing.

Reply:
This method was removed since it wasn't used in the test